jobs/generation/Utilities.groovy

```diff
@@ -66,9 +95,13 @@ class Utilities {
               not {
                 stringsMatch("\${version}", "stable", false)
               }
+            }
```

Comment:
This } is related to `and` above. Previous condition was `!stable && !servicing && !rtm`, why has this changed?

Reply:
Previous code leads to generate just `!rtm`, now it corresponds to `!stable && !servicing && !rtm`